src/ResourceManager/LogicApp/Commands.LogicApp.Test/Commands.LogicApp.Test.csproj

```diff
@@ -0,0 +1,265 @@
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.Azure.TestFramework.1.2.0-preview\lib\net45\Microsoft.Rest.ClientRuntime.Azure.TestFramework.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.WindowsAzure.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.WindowsAzure.Management.4.1.1\lib\net40\Microsoft.WindowsAzure.Management.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Moq">
+      <HintPath>..\..\..\packages\Moq.4.2.1402.2112\lib\net40\Moq.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Net" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Core">
+      <RequiredTargetFramework>3.5</RequiredTargetFramework>
+    </Reference>
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+    </Reference>
+    <Reference Include="xunit, Version=1.9.2.1705, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\xunit.1.9.2\lib\net20\xunit.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="xunit.extensions, Version=1.9.2.1705, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\xunit.extensions.1.9.2\lib\net20\xunit.extensions.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="ScenarioTests\WorkflowAccessKeyTests.cs" />
+    <Compile Include="ScenarioTests\WorkflowController.cs" />
+    <Compile Include="ScenarioTests\WorkflowRunTests.cs" />
+    <Compile Include="ScenarioTests\WorkflowTests.cs" />
+    <Compile Include="ScenarioTests\WorkflowTriggerTests.cs" />
+    <Compile Include="UnitTests\CreateLogicAppTests.cs" />
+    <Compile Include="UnitTests\LogicAppUnitTestBase.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="MSSharedLibKey.snk" />
```

Comment:
[Fixed]

From: Hovsep [mailto:notifications@github.com]
Sent: Monday, February 1, 2016 3:24 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Pankaj Singh Negi <Pankaj.Negi@microsoft.com>
Subject: Re: [azure-powershell] Azure Logic App PowerShell Cmdlets (#1753)


In src/ResourceManager/LogicApp/Commands.LogicApp.Test/Commands.LogicApp.Test.csproj<https://github.com/Azure/azure-powershell/pull/1753#discussion_r51498474>:

> +      <HintPath>..\..\..\packages\xunit.extensions.1.9.2\lib\net20\xunit.extensions.dll</HintPath>

> +      <Private>True</Private>

> +    </Reference>

> +  </ItemGroup>

> +  <ItemGroup>

> +    <Compile Include="Properties\AssemblyInfo.cs" />

> +    <Compile Include="ScenarioTests\WorkflowAccessKeyTests.cs" />

> +    <Compile Include="ScenarioTests\WorkflowController.cs" />

> +    <Compile Include="ScenarioTests\WorkflowRunTests.cs" />

> +    <Compile Include="ScenarioTests\WorkflowTests.cs" />

> +    <Compile Include="ScenarioTests\WorkflowTriggerTests.cs" />

> +    <Compile Include="UnitTests\CreateLogicAppTests.cs" />

> +    <Compile Include="UnitTests\LogicAppUnitTestBase.cs" />

> +  </ItemGroup>

> +  <ItemGroup>

> +    <None Include="MSSharedLibKey.snk" />

Please remove - we will delete it from other projects as well.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1753/files#r51498474>.